components/Header/MessageBar.js

```diff
@@ -20,10 +22,37 @@ const Location = styled.p`
   `};
 `;
 
+const StyledLink = styled.a`
+  margin-left: 2rem;
+  color: ${({ theme }) => theme.colors.white};
+  // text-decoration: underline;
+  border-bottom: solid 1px ${({ theme }) => theme.colors.white};
+
+  &:hover {
+    color: ${({ theme }) => theme.colors.highlight};
+    border-bottom: solid 1px ${({ theme }) => theme.colors.highlight};
+    cursor: pointer;
+  }
+`;
+
 const MessageBar = ({ className }) => {
+  const clickTracking = () => {
+    gtag.event({
+      clientWindow: window,
+      action: 'click',
+      category: 'link button',
+      label: 'message bar link',
+    });
+  };
+
   return (
     <div className={className}>
```

Comment:
I'm not entirely sure this was introduced with your PR, but the background color isn't matching on the div for the message with the background bar. Intentional?

<img width="408" alt="2019-12-27_14-23-51" src="https://user-images.githubusercontent.com/373903/71531496-978c4c80-28b4-11ea-87a2-c6052fa1a3c7.png">

Reply:
Intentional, that is how it is designed. How the site now just a bit more noticeable as this make that area grow.